tests/test_core.py

```diff
@@ -198,11 +203,10 @@ def test_emojize_version():
     assert emoji.emojize("Biking :man_biking: is in 4.0", version=3.0, handle_version=lambda e, data: '<emoji>') == "Biking <emoji> is in 4.0"
     assert emoji.emojize("Biking :man_biking: is in 4.0", version=3.0, handle_version=lambda e, data: data["fr"]) == "Biking :cycliste_homme: is in 4.0"
 
-    def f(emj, data):
+    def f(emj: str, data: Dict[str, str]) -> str:
         assert data['E'] == 5
```

Comment:
This method is a callable for handle_version, and is typed

`handle_version: Optional[Union[str, Callable[[str, Dict[str, str]], str]]] = None` in `core.py`

but this method return None.

What is correct here?

Reply:
Just add a `return ''`